screens/StartScreen.tsx

```diff
@@ -0,0 +1,44 @@
+import { StyleSheet, TextInput, Text, View, Button } from 'react-native';
+import { useState } from 'react';
+import { NavigationContainer } from '@react-navigation/native';
+import { createNativeStackNavigator } from '@react-navigation/native-stack';
+import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
+import { Login } from './Login';
+
+export const StartScreen = () => {
+  const toLoginScreen = () => {};
+  const toSignupScreen = () => {};
+
+  return (
+    <View style={styles.container}>
+      <Text style={styles.heading}>Hello! Let's get started.</Text>
+
+      <View style={styles.button}>
+        <Button onPress={toLoginScreen} title="Login" color="#f2f2f2" />
```

Comment:
Not sure why these buttons are different greys, but you can probably just leave out the colors for now. We'll need to restyle the buttons soon anyways, and we want to avoid using custom hex code strings directly in-line (in the components).